src/components/HomePageComponent.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import CollectRImage from "../images/CollectR.jpg";
+
+export const HomeComponent: React.FC = () => {
+  return (
+    <>
+      <img src={CollectRImage} className="CollectR-logo" alt="CollectR-logo" />
+      <div>
+        <button>
+          <Link to={"/login"}>Login</Link>
+        </button>
+        {/* //Browse button does not do anything upon click yet */}
```

Comment:
When it is clicked in the future, will it bring the user to the search page?

Reply:
Yes, that's correct!